addons/html_editor/static/src/editor/list/list_plugin.js

```diff
@@ -323,7 +305,7 @@ export class ListPlugin extends Plugin {
             }
         }
 
-        const selectionToRestore = this.shared.getEditableSelection();
+        const selectionToRestore = { ...this.shared.getEditableSelection() };
```

Comment:
why clone the selection?

Reply:
He changes some properties of this cloned object a few lines below, so he can call `this.shared.setSelection(selectionToRestore, false);` with it. (see lines 364 - 369)